docs/video-player.md

```diff
@@ -0,0 +1,128 @@
+# `<VideoPlayer/>` component for easygoing videos.
+
+`<VideoPlayer />` is a React component specially designed to work seamlessly with DatoCMS’s [`video` GraphQL query](https://www.datocms.com/docs/content-delivery-api/images-and-videos#videos) that optimizes video streaming for your sites.
```

Comment:
Can we add a short paragraph here clarifying our relationship with Mux, for people who aren't already aware? Just something simple, like:

"To stream videos, DatoCMS partners with [Mux](https://mux.com/), a video CDN that serves optimized streams to your users. Our component is a simple wrapper over [Mux's video player for React](https://docs.mux.com/guides/mux-player-web). It takes care of the details for you, and this is our recommended way to serve optimal videos to your users."